node.gyp

```diff
@@ -33,7 +33,7 @@
     # Windows command length limit or there would be an error.
     # See https://docs.microsoft.com/en-us/troubleshoot/windows-client/shell-experience/command-line-string-limitation
     'library_files': [
-      '<!@(python tools/search_files.py --ext js lib)',
+      '<!@pymod_do_main(search_files --ext js lib)',
```

Comment:
I think we should just simply assign this variable in configure.py, then there is no need for `pymod_do_main`

Reply:
What Now